lib/matplotlib/axes/_axes.py

```diff
@@ -1183,13 +1185,13 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
         linewidths = np.asarray(linewidths)
 
         if len(lineoffsets) == 0:
-            lineoffsets = [None]
+            lineoffsets = [0]
         if len(linelengths) == 0:
-            linelengths = [None]
-        if len(linewidths) == 0:
-            lineoffsets = [None]
+            linelengths = [1]
         if len(linewidths) == 0:
-            lineoffsets = [None]
+            linewidths = [None]
+        if len(linestyles) == 0:
```

Comment:
I understand why to do this for symmetry, but this is a odd special case.  Why would you pass in an empty list to get the default behavior?

Reply:
I guess one could also ask why one would want to pass an empty list to get the default for any of the earlier parameters... (unless there's a compelling reason, we should get rid of that behavior :-))